.gitignore

```diff
@@ -102,3 +103,11 @@ dist
 
 # TernJS port file
 .tern-port
+
+.DS_Store
+
+/public/
+
+venv
+
+.DS_Store
```

Comment:
Lagt inn dobbelt .DS_Store her, men har ikke så mye å si for jeg skal fikse i neste PR